src/views/UserEdit/UserForm.js

```diff
@@ -55,10 +55,6 @@ function validate(values) {
     errors.username = <FormattedMessage id="ui-users.errors.missingRequiredUsername" />;
   }
 
-  if (!values.id && (!values.creds || !values.creds.password) && values.username) {
-    errors.creds = { password: <FormattedMessage id="ui-users.errors.missingRequiredPassword" /> };
-  }
-
```

Comment:
@mkuklis, Is there a test case we should add/update here? The fact that this bug leaked through suggests we don't have adequate coverage for submitting the new-user form. That's pretty crucial functionality!

Reply:
@zburke this bug leaked because this code was never executed (due to the fact that the validation was not functional at all). I do agree that we should probably add more tests to `UserForm`. They do not exist at this point at all.